code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -616,6 +621,16 @@
 	icon_open = "smw_hoodie_open"
 	icon_closed = "smw_hoodie"
 
+/obj/item/clothing/suit/storage/toggle/redandblackjacket
+    name = "red and black jacket"
+    desc = "A cool red and black jacket to keep you stylish and cozy."
+    icon_state = "redandblackjacket"
+    icon_open = "redandblackjacket_open"
+    icon_closed = "redandblackjacket"
+    min_cold_protection_temperature = T0C - 20
+    cold_protection = UPPER_TORSO|LOWER_TORSO|ARMS

```

Comment:
copied code for a hoodie, figured I should keep it since it's a jacket and makes sense. Want it removed?